src/main/java/hudson/plugins/tics/TicsPublisher.java

```diff
@@ -97,7 +99,7 @@ public boolean getFailIfQualityGateFails() {
     }
 
     @Override
-    public void perform(@Nonnull final Run<?, ?> run, @Nonnull final FilePath workspace, @Nonnull final Launcher launcher, @Nonnull final TaskListener listener) throws IOException, RuntimeException, InterruptedException {
+    public void perform(@Nonnull final Run<?, ?> run, @Nonnull final FilePath workspace, @NonNull final EnvVars envvars, @Nonnull final Launcher launcher, @Nonnull final TaskListener listener) throws IOException, RuntimeException, InterruptedException {
```

Comment:
Why is envvars added here? It looks like it is not being used.

Reply:
That is to fix deprecation warning in a Jenkins API. The version of perform() without EnvVars is deprecated.